tests/std/tests/P1206R7_vector_from_range/test.cpp

```diff
@@ -124,6 +124,40 @@ void test_lvalue_forward_list() {
     }
 }
 
+struct counted_item {
+    static int count;
+
+    counted_item() {
+        ++count;
+    }
+
+    counted_item(const counted_item&) {
+        ++count;
+    }
+
+    counted_item& operator=(const counted_item&) = default;
+
+    ~counted_item() {
+        --count;
+    }
+};
+
+int counted_item::count = 0;
+
+void test_vso1591034() {
+    // _Uninitialized_copy was using incorrectly using memcpy for types that are
+    // trivially assignable but not trivially constructible.

```

Comment:
Suggestion: It would be good to `static_assert` these important properties of the test type.